Source/WebCore/html/parser/HTMLPreloadScanner.cpp

```diff
@@ -123,26 +123,25 @@ class TokenPreloadScanner::StartTagScanner {
             return;
         
         for (auto& attribute : attributes) {
-            AtomString attributeName(attribute.name);
-            AtomString attributeValue(attribute.value);
-            processAttribute(WTFMove(attributeName), WTFMove(attributeValue), pictureState);
+            auto knownAttributeName = AtomString::lookUp(attribute.name.data(), attribute.name.size());
```

Comment:
It can become null AtomString, is it OK?

Reply:
Yes, if it’s null then it won’t be equal to any of the atom string constants we compare it against. That’s all we ever do with this name, compare it with constants.